code/modules/mob/living/carbon/brain/death.dm

```diff
@@ -3,7 +3,7 @@
 		return
 	stat = DEAD
 
-	if(!gibbed && container && istype(container, /obj/item/device/mmi))//If not gibbed but in a container.
+	if(!gibbed && container)//If not gibbed but in a container.

```

Comment:
Well, that doesn't guarantee it will be an MMI; it just casts it automatically due to *assuming* it is, but it's still dynamically typed. However, we probably want to runtime in that edge case either way, so I guess not checking isn't that big of a deal.